HURUmap Tanzania update Story Showcase images and Partner Logos

josephluvanda opened the pull request:
## Description

This change updates story showcase and images, partner logos and adds 'Number of desks' dataset(Disabled, pending data discrepancy fix)

Fixes # 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update


## Screenshots
![screenshot-127 0 0 1-8000-2017-12-20-12-35-15-007](https://user-images.githubusercontent.com/1044020/34201324-30548fba-e585-11e7-8e3e-adb3f1b6fcb0.png)



## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have  performed a self-review of my own code
- [x] I have commente] my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

josephluvanda committed:
Update Showcase images and partner logos

josephluvanda committed:
Add Number of Desks dataset

DavidLemayian commented on hurumap/static/css/hurumap.css:
```diff
@@ -0,0 +1,235 @@
+  padding: 3px 6px 1px !important; }
+
+.how-it-works img {
+  max-width: 70px; }
```
What's this file doing here?

DavidLemayian commented on hurumap_tz/static/css/hurumap.css:
```diff
@@ -0,0 +1,234 @@
+  padding: 3px 6px 1px !important; }
+
+.how-it-works img {
+  max-width: 70px; }
```
What's this file for?

DavidLemayian commented on hurumap_tz/static/css/hurumap.scss:
```diff
@@ -0,0 +1,361 @@
+  img {
+    max-width: 70px;
+  }
+}
```
What's this file for?

josephluvanda commented:
@DavidLemayian, I added new showcase images and they were being pulled on one side. 
I removed  `background-position: center center;` in hurumap.scss and ran `python manage.py compilescss` to position the new showcase images. Showcase with `background-position: center center;`

![screenshot-127 0 0 1-8000-2018-01-09-09-10-58-933](https://user-images.githubusercontent.com/1044020/34707184-1f3fe6a0-f51d-11e7-8290-e6b0c89e45c7.png)


  @

DavidLemayian commented:
@phillipahereza Could you take a look at this to align best way to achieve these customisations?

phillipahereza commented:
@darkdant3 change `background-size: cover` to `background-size: contain`
<img width="1169" alt="screen shot 2018-01-11 at 12 35 11 pm" src="https://user-images.githubusercontent.com/13383422/34818469-eb39983c-f6cb-11e7-8f8b-244c0d70ce4a.png">

phillipahereza commented:
@darkdant3 the easiest solution in my opinion would be to change the showcase template and use the `img` tag instead of background images since the showcase images are the same size and style.
Secondly, the `hurumap.css` file is automatically generated so be careful when adding new files for git to track. I'd suggest you add the specific files instead of the global add or add the generated files to your `gitignore`

josephluvanda committed:
Remove auto generated css + change showcase images to use img tags

DavidLemayian commented on hurumap_tz/templates/profile/profile_detail.html:
```diff
@@ -358,8 +358,12 @@ <h1>Primary School Teachers</h1>
+          <div class="column-half">
+              {% include 'profile/_blocks/_stat_list.html' with stat=primary_school_desks.number_of_desks stat_type='number' %}
+          </div>
+      </section>-->
```
Would this be needed if it's commented out?

josephluvanda replied:
Yes, It's pending a data discrepancy fix and some new stats

DavidLemayian commented:
Good job on the update @darkdant3. @phillipahereza please approve if ready to merge.
